<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">
   <item>
    <layout class="QVBoxLayout" name="sidebar_layout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>50</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>50</number>
     </property>
     <item>
      <widget class="QPushButton" name="main_page_button_timetable">
       <property name="font">
        <font>
         <family>Noto Sans CJK TC</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Home</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="login_history_button_timetable">
       <property name="font">
        <font>
         <family>Noto Sans CJK HK</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Login History</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="logout_button_timetable">
       <property name="font">
        <font>
         <family>Noto Sans CJK HK</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Logout</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="content_layout">
     <property name="leftMargin">
      <number>50</number>
     </property>
     <property name="topMargin">
      <number>50</number>
     </property>
     <property name="rightMargin">
      <number>50</number>
     </property>
     <property name="bottomMargin">
      <number>50</number>
     </property>
     <item>
      <widget class="QLabel" name="welcome_message_label_timetable">
       <property name="font">
        <font>
         <family>Noto Sans CJK HK</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Welcome [Name]</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QTableWidget" name="timetable_table">
       <row>
        <property name="text">
         <string>Date</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>07:00</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>07:30</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>08:00</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>08:30</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>09:00</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>09:30</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>10:00</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>10:30</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>11:00</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>11:30</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>12:00</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>12:30</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>13:00</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>13:30</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>14:00</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>14:30</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>15:00</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>15:30</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>16:00</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>16:30</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>17:00</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>17:30</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>18:00</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>18:30</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>19:00</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>19:30</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>20:00</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>20:30</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>21:00</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>21:30</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>22:00</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>22:30</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>Mon</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Tue</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Wed</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Thu</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Fri</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Sat</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Sun</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
